lib/v2/vcb-s/index.js

```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+const rootUrl = 'https://vcb-s.com';
+const postsAPIUrl = `${rootUrl}/wp-json/wp/v2/posts`;
+
+module.exports = async (ctx) => {
+    const limit = ctx.params.limit ?? 7;
```

Comment:
Please use the common parameter `ctx.query.limit` docs: https://docs.rsshub.app/parameter.html